TACC/Core-Portal
# Bug/WP-321: Removing oversized attachments from tickets

## Overview

Users are able to submit tickets when a problem arises. Users can submit files along with these tickets as long as they are less than 3 MB, but when a user adds a file that's too big, it cannot be removed unless the Ticket modal gets closed and reopened.

## Related

* [WP-321](https://tacc-main.atlassian.net/browse/WP-321)

## Changes

When a user attaches an oversized file to a ticket, I added a Remove button to the end of that row. Users can now click on that button to remove the oversized file.

## Testing

1. Log in to the Portal and go to the Dashboard.
2. Click on New Ticket.
3. Attach a file that is more than 3 MB to the ticket. You'll see a warning that the file exceeds the file size limit.
4. Click on the Remove button on that row to remove the oversized file.

## UI

![image](https://github.com/user-attachments/assets/b7aff054-969f-4b7d-aaea-58278ad7f164)

## Notes
